src/access/acp.test.ts

```diff
@@ -2618,3 +2619,567 @@ describe("getActorAccess", () => {
     });
   });
 });
+
+describe("internal_getActorAccessAll", () => {
+  it.each([acp.agent, acp.group])(
+    "returns an empty map if no individual %s is given access",
+    (actor) => {
+      const resourceWithAcr = mockResourceWithAcr(
+        "https://some.pod/resource",
+        "https://some.pod/resource?ext=acr",
+        {
+          policies: {},
+          memberPolicies: {},
+          acrPolicies: {},
+          memberAcrPolicies: {},
+        }
+      );
+      expect(internal_getActorAccessAll(resourceWithAcr, actor)).toStrictEqual(
+        {}
+      );
+    }
+  );
+
+  it("does not return access given to individual agents for groups", () => {
+    const resourceWithAcr = mockResourceWithAcr(
+      "https://some.pod/resource",
+      "https://some.pod/resource.acr",
+      {
+        policies: {
+          "https://some.pod/resource.acr#policy": {
+            anyOf: {
+              "https://some.pod/resource.acr#rule": {
+                [acp.agent]: ["https://some.pod/profile#agent"],
+              },
+            },
+            allow: {
+              append: true,
```

Comment:
Btw, no biggie (I haven't stuck to this in earlier unit tests either), but generally I think it is a good idea to only vary one property that you're explicitly testing and leave the rest exactly the same. So if you want to test whether it also works for `append`, that would be a separate test. This makes the tests more predictable in the sense that some tests won't suddenly break on an unrelated change whereas others will.

Again, no need to fix this here, but maybe something to keep in mind for the future.